tests/integration/live_tests.py

```diff
@@ -0,0 +1,112 @@
+"""
+Live tests for pyltc.
+
+We execute actual tc commands and then launch iperf server and client
+and measure the badwith.
+
+"""
+from os.path import abspath, normpath, dirname, join as pjoin
+REPO_ROOT = normpath(abspath(pjoin(dirname(__file__), "..", "..")))
+
+import sys
+if not REPO_ROOT in sys.path:
+    sys.path.append(REPO_ROOT)
+
+import unittest
+from pyltc.main import pyltc_entry_point
+from pyltc.plugins.util import parse_branch
+from pyltc.util.rates import convert2bps
+from tests.util.base import LtcLiveTargetRun
+from tests.util.iperf_proc import TCPNetPerfTest
+
+
+DURATION = 2
+VERBOSITY = False
+MAX_FREE_RATE_TOLERANCE = 0.75
+MAX_SHAPED_TOLERANCE = 0.25
```

Comment:
Would be good if you comment these parameters, like `MAX_FREE_RATE_TOLERANCE = 0.75  # the ratio of measured / expected rate when no shaping is applied that we tolerate as non-suspicious.`